person.rb

```diff
@@ -24,5 +25,9 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Kindly refactor this method based on the feedback provided in the book class above.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```

Reply:
Thank you @DammyShittu, you are really helpful with these solutions. Appreciate